spec/rubocop/cop/rspec/is_expected_specify_spec.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::IsExpectedSpecify, :config do
+  it 'registers an offense when using `specify` and one-liner style' do
+    expect_offense(<<~RUBY)
+      specify { is_expected.to be_truthy }
+      ^^^^^^^ Use `it` instead of `specify`.
+      specify { are_expected.to be_falsy }
+      ^^^^^^^ Use `it` instead of `specify`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      it { is_expected.to be_truthy }
+      it { are_expected.to be_falsy }
+    RUBY
+  end
+
+  it 'registers an offense when using `specify` and one-liner with metadata' do
+    expect_offense(<<~RUBY)
+      specify "pending", :pending
+      ^^^^^^^ Use `it` instead of `specify`.
```

Comment:
I’m uncertain if we should do this. Why? There’s no is_expected. Also the docstring can be descriptive. And there’s another cop that would suggest adding a reason for pending.
I think we can safely ignore this.

This would also allow to simplify the guard clause into two more focused lines.

Reply:
Uh, sorry 😨 I was a little confused. It's right not to offense that.